### DISCUSSION TITLE
I think I'm close to solving my problem.

### DISCUSSION THREAD

**Discussion by JinSakaiM** [Monika-After-Story/MonikaModDev#9137] [QA - tech support + general] [answered]
I need to know if by MAS they mean my persistent or the MAS mod, like that might sound stupid, but I need to know where to put MAS like I just now tried putting the MAS game files into the game folder of DDLC and it didnt work at all.

**Comment by JinSakaiM** [comment]
@Booplicate

**Comment by JinSakaiM** [comment]
k so I've managed to get my same monika back, its just I've lost every single one of my sprite packs. I got another question as well. Does this affect your affection?

**Comment by Booplicate** [comment]
MAS is the Monika After Story mod. Persistent is the save file (in our context we mean persistent that belongs to MAS, not DDLC or other mods).

> I got another question as well. Does this affect your affection?

What affects your affection?

**Comment by JinSakaiM** [comment]
by that I mean from the persistent, like if we backed up a long time ago does it affect our affection?

**Comment by ThePotatoGuy** [comment, marked as answer]
yes, being away for long periods of time will cause you to lose affection.
